src/runtime_src/driver/xclng/drm/xocl/subdev/mb_scheduler.c

```diff
@@ -2402,56 +2428,59 @@ static void destroy_client(struct platform_device *pdev, void **priv)
 	unsigned int	timeout_loops = 20;
 	unsigned int	loops = 0;
 	int pid = pid_nr(task_tgid(current));
-	unsigned bit;
+	unsigned int bit;
 	struct ip_layout *layout = XOCL_IP_LAYOUT(xdev);
 
 	bit = layout
-		? find_first_bit(client->cu_bitmap, layout->m_count)
-		: MAX_CUS;
+	  ? find_first_bit(client->cu_bitmap, layout->m_count)
```

Comment:
not required to use tab for indent?

Reply:
Guess not.  It's part of the expression, not a new statement; maybe that's why.